### DISCUSSION TITLE
How to read / get signature field / area

### DISCUSSION THREAD

**Discussion by arcontechnologies** [pymupdf/PyMuPDF#1734] [Looking for help]
Dear,
I'm able to get the widgets part of my pdf but I can't get the signature part. I tried by getting it as image but nothing was detected. Do you have any idea how I can acheive this. Thanks for your help

![image](https://user-images.githubusercontent.com/17960609/170940960-6b8b3d75-a3fd-4e92-9724-5a575e02cbc2.png)

**Comment by JorjMcKie** [comment]
This is not supported in PyMuPDF.
But you can use `doc.convert_to_pdf()` to make another PDF of your doc. You can also restrict the function to one page.
Anyway, in the result annotations and fields will be converted to regular PDF content, so you will find the signatures as normal images, which you can extract.

Unfortunately, this will not always work, because MuPDF does not support certain fonts ...

**Reply by arcontechnologies** [reply]
Thanks for the tip. I'll give it a try
